concourse/scripts/publish.sh

```diff
@@ -17,7 +17,7 @@ then
   npm run build:package
   echo "Publishing a new version..."
   echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc
-  npm publish
+  npm publish --tag beta
```

Comment:
Is this change necessary?

Reply:
It was in the `v5` branch that I rebased from. Can one of the other reviewers confirm that it belongs there?

Reply:
If it was in the `v5` branch, it shouldn't be in your PR. You're making a documentation-only change, so the only modifications should be to the documentation.

Reply:
OK, change to publish.sh is undone.